generate-deb.sh

```diff
@@ -432,16 +432,16 @@ Source: dropbox
 Section: gnome
 Priority: optional
 Maintainer: Dropbox <support@dropbox.com>
-Build-Depends: cdbs, debhelper (>= 9), build-essential, libnautilus-extension-dev (>= 3.10.1), libglib2.0-dev (>= 2.40), python-gi (>= 3.12), python-docutils
+Build-Depends: cdbs, debhelper (>= 9), build-essential, libnautilus-extension-dev (>= 3.10.1), libglib2.0-dev (>= 2.40), python-gi (>= 3.12), python-docutils, python3-gi (>= 3.12), python3-docutils
```

Comment:
Hmm, `python-gi` shouldn't be needed anymore... and `python3-docutils` isn't needed **yet** as `rst2man.py` is still Python 2.

Reply:
Thanks @monsta Yes, it should be `python-docutils` for now.

Reply:
Fixed by #69